src/quo2/components/wallet/account_overview/view.cljs

```diff
@@ -0,0 +1,69 @@
+(ns quo2.components.wallet.account-overview.view
+  (:require [quo2.components.icon :as icons]
+            [quo2.components.markdown.text :as quo2]
+            [quo2.foundations.colors :as colors]
+            [react-native.core :as rn]
+            [utils.i18n :as i18n]
+            [quo2.components.wallet.account-overview.style :as style]))
+
+(defn account-overview
+  [{:keys [account-name current-value bottom-time-text bottom-time-to-text show-details?
```

Comment:
Hey @ibrkhalil, we want to move away from creating our own api names for components - it makes it very difficult to find the correct configuration when using the component on a given page and then also makes it arduous to verify the component is using the correct configuration. 
the api should be mostly reflecting this:
<img width="329" alt="Screenshot 2023-07-21 at 13 41 57" src="https://github.com/status-im/status-mobile/assets/22799766/0ddd6b2e-2382-45bc-919e-5d537a3150f5">

Reply:
By loading state you mean the skeleton one?

Reply:
Got it, I missed it.
I'll ping you when I need another look
Thanks Jamie

Reply:
Please take another look when you have time Jamie